docs/docs/adrs/adr-007-pause-unbonding-on-eqv-prop.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 2
+title: ADR Template
+---
+# ADR 007: Pause validator unbonding during equivocation proposal
+
+## Changelog
+* 2023-05-16: Initial Draft
+
+## Status
+
+Proposed
+
+## Context
+
+(Copied from [#747](https://github.com/cosmos/interchain-security/issues/747))
+
+Currently, if an equivocation slashing proposal is created after more than one
+week has passed since the equivocation, it is possible that the validator in
+question could unbond and get away without being slashed, since the unbonding
+period is 3 weeks, and the voting period is 3 weeks. For this reason, it might
+be good to pause unbondings for validators named in an equivocation slashing
+proposal until the proposal's voting period is over.
+
+## Decision
+
+### How
+
+Pausing the unbonding period is already possible thanks to the changes in the
+`staking` module of the cosmos-sdk:
+- `stakingKeeper.PutUnbondingOnHold` pauses an unbonding period
+- `stakingKeeper.UnbondingCanComplete` unpauses an unbonding period
+
+These methods use a counter under the hood, which means we can use them without 
+conflicts with the *Completion of Unbonding Operations* system. Giving an
+unbonding period (already paused by definition because of the *Completion of
+Unbonding Operations* system), an additional pause will just increase the
+counter, so when this unbonding period has reached its maturity on provider
+and all consumer chains, it will remain paused.
+
+### When pause
+
+The unbonding period (if there is any unbonding) should be paused once an
+equivocation proposal enters the voting period. For that, the `gov` module's
+hook `AfterProposalDeposit` can be used. 
+
+If the hook is triggered with a an equivocation proposal in voting period, then
+for each equivocation of the proposal, the related unbonding operations of the
```

Comment:
> the related unbonding operations of the related validator must be paused.

This needs to be described more precisely. For example, unbonding operations that are no longer waiting for this particular consumer chain should probably not be put on hold. Also, if the infraction occurred after the unbonding operation was initiated, there is no reason to pause the unbonding.

Reply:
> For example, unbonding operations that are no longer waiting for this particular consumer chain should probably not be put on hold.

Actually, for this particular case, I wonder how this is possible. Considering the protocol, how an unbonding period can end on a consumer chain *before* the provider chain ? 

Or maybe when the unbonding period is deliberately shorter between consumer and provider ?

Reply:
For 

> Also, if the infraction occurred after the unbonding operation was initiated,

Fixed dd353c29df93d8e7940098516bce5da9fe0cee64